### PR TITLE
resolving #781

### PR DESCRIPTION
**Pull request by Pratyush-Saxena**
Icons added ..replacing Text-links in footer

### CODE REVIEW

**Comment by nzakas** [comment]
Thanks for taking a look at this. If you could please take a look at [adding the email you used](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user)  for this commit, we need to get the CLA signature correct.

**Comment by nzakas** [comment]
This looks like a good first pass. Please take a look at https://deploy-preview-783--eslint.netlify.app/, which previews the site with the change. It looks like the changes are a bit jumbled on mobile. Can you take a look?

**Comment by Pratyush-Saxena** [comment]
> This looks like a good first pass. Please take a look at https://deploy-preview-783--eslint.netlify.app/, which previews the site with the change. It looks like the changes are a bit jumbled on mobile. Can you take a look?

Well i didn't find anything Jumbeled ....on my mobile
processed
is it that ....Open Js Foundation logo ..you are talking ablout?

**Comment by nzakas** [comment]
That's what I'm seeing, too. Removing the OpenJS Foundation logo will help. I'd also like to see "Edit this page" either on a separate line from the icons or have the icons aligned to the right of the page to separate them from "Edit this page" (only on mobile, of course)

**Comment by Pratyush-Saxena** [comment]
> That's what I'm seeing, too. Removing the OpenJS Foundation logo will help. I'd also like to see "Edit this page" either on a separate line from the icons or have the icons aligned to the right of the page to separate them from "Edit this page" (only on mobile, of course)

@nzakas Pls have a look now

**Comment by nzakas** [comment]
Much better! The only thing we need now is to replace the “OpenJs Foundation” link with the full URL (as we currently have on the site).

And please don’t forget to address the CLA email issue. 

Thanks!

**Comment by Pratyush-Saxena** [comment]
@nzakas is it all done now....?

File: .theia/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.autoSave": "on"
```
Comment (nzakas) [inline .theia/settings.json:2]: I think this was included by mistake. Can you delete this file?

**Review by nzakas — CHANGES_REQUESTED** [review]
Looks good! Just looks like a file was included by accident. We also still need you to electronically sign the CLA.

**Review by Pratyush-Saxena — COMMENTED** [review]
reviewed

**Comment by Pratyush-Saxena** [comment]
> Looks good! Just looks like a file was included by accident. We also still need you to electronically sign the CLA.

i have already signed it ....
cla

**Comment by nzakas** [comment]
See the original comment: https://github.com/eslint/website/pull/783#issuecomment-690144168

You most likely have a commit that was created using a different email address.

**Comment by Pratyush-Saxena** [comment]
> See the original comment: [#783 (comment)](https://github.com/eslint/website/pull/783#issuecomment-690144168)
> 
> You most likely have a commit that was created using a different email address.

@nzakas  so what should i do now ...pls tell me in a procedural way..

**Comment by nzakas** [comment]
See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user for how to update the email address on your commit. It looks like the first commit in this PR is not linked to your GitHub account. That is the one to fix.

Also, please don't forget to remove `.thea/settings.json` from the PR.

**Comment by Pratyush-Saxena** [comment]
I think i should creat a new pr ...

**Comment by Pratyush-Saxena** [comment]
@nzakas  pls head over to #790

**Pull request closed without merging**